components/observatorium.libsonnet

```diff
@@ -49,8 +50,8 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
           local cfg = self,
           name: obs.config.name + '-' + cfg.commonLabels['app.kubernetes.io/name'] + '-' + hashring.hashring,
           namespace: obs.config.namespace,
-          replicas: 3,
-          replicationFactor: 3,
+          replicas: 1,
```

Comment:
:+1:  this will bring down our resource utilization for the tests. I wonder if we should make tests adjustments in a new `environments/test` environment. So that `environments/dev` and `environments/base` inherit the higher replica counts by default. we can do that in a follow up though